core/field_dropdown.js

```diff
@@ -300,6 +296,10 @@ Blockly.FieldDropdown.prototype.setText = function(text) {
   this.updateTextNode_();
 
   if (this.textElement_) {
+    // Update class for dropdown text.
```

Comment:
Ok, it's because updateTextNode_ sets the class every time also, in case it becomes (non)-editable. Seems like we should leave as-is?